Example/ViewController.swift

```diff
@@ -89,6 +90,25 @@ class ViewController: TableViewController {
     }
 }
 
+extension TableViewController: UITableViewDelegate {
+    // MARK: - UIScrollViewDelegate example functions
+    public func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
+        // You can get UIScrollViewDelegate functions forwarded, even though the `DataSource` instance is the true delegate
+        // ...
+    }
+    
+    // MARK: - UITableViewDelegate example functions
+    public func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
```

Comment:
I can see this item being useful...

But didSelect is covered as part of Static closure handling, and scrollViewWillBeginDragging can be observed with a observer?

Reply:
The functions here are not comprehensive regarding what methods will get forwarded, they are just meant to demonstrate this functionality. They all get forwarded, including any potential future `UIScrollViewDelegate` functions. The only ones that do not are the ones that Static handles completely where it doesn't make sense to forward.

I don't think all `UIScrollViewDelegate` methods can be handled with observers. Developers are more familiar with `UIScrollViewDelegate` & `UITableViewDelegate` methods than they are with observing those same changes. This is why Static has at least 3 tickets around requesting this functionality. If the code only forwarded the calls that can't be done with observers, that would obviously create more confusion.

Reply:
understood